ee/dtr/admin/install/index.md

```diff
@@ -82,8 +82,7 @@ DTR and UCP must not be installed on the same node, due to the potential for res
 
 2. Select **Shared Resources > Stacks** in the left navigation pane. DTR should be listed as a stack.
 
-3. Verify that DTR is accessible from the browser by entering the DTR IP address or FQDN on the address bar. As an [HSTS (HTTP Strict-Transport-Security)
-header](https://en.wikipedia.org/wiki/HTTP_Strict_Transport_Security) is included in all API responses, make sure to specify the FQDN (Fully Qualified Domain Name) of the DTR instance prefixed with `https://` or the web interface may fail to load.
+3. Verify that DTR is accessible from the browser by entering the DTR IP address or Fully Qualified Domain Name (FQDN) in the address bar. As an [HSTS (HTTP Strict-Transport-Security) header](https://en.wikipedia.org/wiki/HTTP_Strict_Transport_Security) is included in all API responses, make sure to specify the FQDN of the DTR instance prefixed with `https://` or the web interface may fail to load.
```

Comment:
What about "Verify that you can see the DTR UI in your web browser by entering the DTR IP address, or the Fully Qualified Domain Name (FQDN) prefixed with 'https://". 
I almost think we shouldn't talk about this, it's like that with all web sites that require https so we're not teaching them about Docker. Leave it up to you as it's a judgment call.